Extension/lib/prefs.js

```diff
@@ -48,7 +48,11 @@ var Prefs = (function () {
             }
             return browser;
         })(),
-        betaYandex: navigator.userAgent.indexOf("Yowser") >= 0
+        betaYandex: navigator.userAgent.indexOf("Yowser") >= 0,
+        chromiumVersion: (function () {
```

Comment:
We could simply rename it to "version" as this field may be useful for other browsers